extensions/wikia/FounderEmails/events/FounderEmailsRegisterEvent.class.php

```diff
@@ -40,7 +40,7 @@ public function process( Array $events ) {
 		return true;
 	}
 
-	public static function register( User $user ) {
+	public static function registerUser( User $user = null ) {
```

Comment:
Do we want a default value here?

Reply:
fixed in https://github.com/Wikia/app/pull/13954/commits/6a0ec4e6f86d3f04545538e31b59bbfa42e6426a